dashboard/app/models/pd/workshop.rb

```diff
@@ -200,6 +200,7 @@ def self.send_automated_emails
   def self.process_ended_workshop_async(id)
     workshop = Pd::Workshop.find(id)
     raise "Unexpected workshop state #{workshop.state}." unless workshop.state == STATE_ENDED
+    workshop.update!(processed_at: Time.zone.now)
```

Comment:
Should this follow `send_exit_surveys` so that `processed_at` isn't set if there are issues in that method?